packages/orbit-components/src/Tooltip/__tests__/index.test.jsx

```diff
@@ -15,9 +16,16 @@ jest.mock("../../hooks/useMediaQuery", () => {
 describe("Tooltip", () => {
   it("it should render Tooltip", () => {
     const content = "Write some message to the user";
+    const onShow = jest.fn();
 
-    render(<Tooltip content={content}>kek</Tooltip>);
+    render(
+      <Tooltip content={content} onShow={onShow}>
+        kek
+      </Tooltip>,
+    );
 
     expect(screen.getByText("kek")).toBeInTheDocument();
+    userEvent.hover(screen.getByText("kek"));
+    expect(onShow).toHaveBeenCalled();
```

Comment:
Don't you think this deserves its own test? Separate the testing responsibilities so that when one fails it is for (hopefully) just one reason.

Reply:
Good question. It says "it should render Tooltip", so I guess it's ok to have it like that :)

Reply:
Yes, when I suggested a separate test I was thinking of something like "it fires onShow". I'm ok with leaving them together, I was just considering more granular tests, but I think both work.